netdata-installer.sh

```diff
@@ -1,18 +1,19 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
I thought that `#!/usr/bin/env bash` is more portable than `#!/bin/bash` and we certainly want portability for the `netdata-installer.sh` since it's a 'user facing' script.

```suggestion
#!/usr/bin/env bash
```

Reply:
Also, if one's upgraded their default bash installation (e.g. brew on Mac), the `env` is needed to get the right version.

Reply:
Okay I'm sold :) Mostly on the macOS / Brew argument.

FWIW most systems have their shells installed to `/bin` But 🤷‍♂️